doc/source/getting_started/comparison/comparison_with_sql.rst

```diff
@@ -65,24 +65,9 @@ Filtering in SQL is done via a WHERE clause.
 
     SELECT *
     FROM tips
-    WHERE time = 'Dinner'
-    LIMIT 5;
```

Comment:
This is the only change of substance: left out the `LIMIT` and the `.head(5)` below since they aren't needed, nor relevant to filtering.